### DISCUSSION TITLE
Can't login after last update

### DISCUSSION THREAD

**Discussion by makovez** [lxqt/lxqt#1980] [General]
I have updated lxqt to last version and now im not able to login. As soon as i enter the password and click on the login i get a black screen and right after I get back to the login page. I have tried to login in the terminal and run startx and the desktop loaded succesfully. What might be the cause of this? How can i fix it?

**Comment by tsujan** [comment]
> What might be the cause of this?

An incomplete upgrade or bad builds.

Which distro? Manual upgrade or distro upgrade?

**Reply by tsujan** [reply]
> I have tried to login in the terminal and run startx and the desktop loaded succesfully.

Then, the problem may be in the login manager. sddm, lightdm, gdm,...?

**Reply by makovez** [reply]
normal upgrade.
i don't know, how can i understand that?

**Reply by tsujan** [reply]
> i don't know, how can i understand that?

 1. Did you upgrade by using the package manager of your distro or by compiling from sources?
 2. Which distro?
 3. Which login manager?

**Reply by makovez** [reply]
1. by the package manager
2. lubuntu
3. sddm

**Reply by tsujan** [reply]
OK. I think you may want to tell Ubuntu about the problem.

A previous version of SDDM had a hang problem on login but I don't know if the problem you've encountered is related to that (NOTE: SDDM isn't an LXQt component). Anyway, Ubuntu maintainers might need your report at Launchpad.

**Reply by stefonarch** [reply]
Try creating a new user and if login works for him the problem is in your home/configuration.
Upgraded lxqt-packages on Lubuntu to which version?
